pkg/sliceutils/equal.go

```diff
@@ -0,0 +1,14 @@
+package sliceutils
+
+// Equal checks if the two given slices are equal.
+func Equal[T comparable](a, b []T) bool {
```

Comment:
Another function does `slice1` and `slice2`. Maybe we should converge on a standard?

Reply:
I used `slice1` and `slice2` here and also in `Difference`.